### ISSUE TITLE
Entrypoint for docker-run task

### ISSUE THREAD

**Issue by brianbolt** [microsoft/vscode-docker#2725]
As I understand it, the entrypoint for python debugging is overridden to just "python" automatically to keep the container running.  Then if my "python" module is flask, it will execute flask with the arguments.  This doesn't allow for any room to set a dynamic "LD_LIBRARY_PATH" before my flask app starts.  Typically we execute a shell script as our entrypoint, which then sets the LD_LIBRARY_PATH (based on runtime library mapped into the container) in a .env file which the flask app picks up.

My main question is How can I achieve a dynamic LD_LIBRARY_PATH like I get when I can control the entrypoint?

Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 20.3.0

Name: Docker
Id: ms-azuretools.vscode-docker
Description: Makes it easy to create, manage, and debug containerized applications.
Version: 1.10.0
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker

**Comment by bwateratmsft** [comment]
Hm, it looks like we missed the `entrypoint` option in the package.json schema for `docker-run` tasks, but it is an option the code should recognize. As a rule, we respect user input over defaults with very few exceptions.

This may give a yellow warning underline because of the above mistake on schema, but you can ignore it.
```json
        {
            "type": "docker-run",
            "label": "docker-run: debug",
            "dependsOn": [
                "docker-build"
            ],
            "dockerRun": {
                "entrypoint": "// Your entrypoint here"
            },
            "python": {
                "args": [
                    "runserver",
                    "0.0.0.0:8000",
                    "--nothreading",
                    "--noreload"
                ],
                "file": "manage.py"
            }
        },
```

**Comment by brianbolt** [comment]
Thank you @bwateratmsft That helps a ton.  Got it working with your tip.

**Comment by bwateratmsft** [comment]
@philliphoff I now recall that in the past we intentionally left `entrypoint` out of package.json, in favor of `command`. Python, unlike Node and .NET, has this [line](https://github.com/microsoft/vscode-docker/blob/main/src/tasks/python/PythonTaskHelper.ts#L107):
```typescript
runOptions.entrypoint = runOptions.entrypoint || 'python';
```

Node and .NET leave it undefined, so if the user provides `command` (which _is_ defined in package.json), it should override. I think what we actually need to do is remove that line from PythonTaskHelper.ts. Thoughts?

**Comment by bwateratmsft** [comment]
Need to change this; it's breaking down debugging.

**Comment by bwateratmsft** [comment]
This is now released in Docker extension version 1.11.0.
